en/docs/get-started/open-packages.md

```diff
@@ -1,16 +1,16 @@
 # Open Ballerina packages
 
-You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [Multi-Root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of VS Code to work on serverl related Ballerina Packages at the same time. 
+You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [multi-Root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of the VS Code editor to work on serveral related Ballerina packages at the same time. 
```

Comment:
Is the link for `multi-root workspaces` correct? Shouldn't it be https://code.visualstudio.com/docs/editor/multi-root-workspaces